modules/tinymce/src/plugins/table/main/ts/core/Util.ts

```diff
@@ -37,14 +37,21 @@ const addSizeSuffix = function (size: string) {
   return size;
 };
 
-const removeDataStyle = (table) => {
+const removeDataStyle = (table: Element<HTMLElement>) => {
   const dataStyleCells = SelectorFilter.descendants(table, 'td[data-mce-style],th[data-mce-style]');
   Attr.remove(table, 'data-mce-style');
   Arr.each(dataStyleCells, function (cell) {
     Attr.remove(cell, 'data-mce-style');
   });
 };
 
+const getRawWidth = (editor: Editor, elm: HTMLElement) => {
+  const raw = editor.dom.getStyle(elm, 'width') || editor.dom.getAttrib(elm, 'width');
+  return Option.from(raw).filter((s) => s.length > 0);
+};
+
+const isPercentage = (value: string) => /(\d+(\.\d+)?)%/.test(value);
```

Comment:
Is this unit tested?

Reply:
The editor team have never done UnitTests for this if covered by another test larger test. Possibly something we should be doing though, so I'll add one for this.